packages/caravan-fees/src/constants.ts

```diff
@@ -0,0 +1,11 @@
+import BigNumber from "bignumber.js";
+
+export const DEFAULT_RBF_SEQUENCE = 0xfffffffd;
+export const MIN_FEE_RATE = 1;
+export const MAX_FEE_RATE = 5000;
+export const DEFAULT_FEE_INCREASE_THRESHOLD = 1.1;
+export const MIN_DUST_AMOUNT = new BigNumber(546);
+export const SATS_PER_BTC = new BigNumber(100000000);
+export const DUST_THRESHOLD = 546; // in satoshis
```

Comment:
Why is this a number but `MIN_DUST_AMOUNT` a BN? 

Also, I wonder if this should be `DEFAULT_DUST_THRESHOLD` as dust really should be a dynamic value that is reactive to the fee market. Granted, in many places and code bases, it is hard coded, but I think allowing for flexibility is merited. I think the functions should also support optional dust thresholds so that a consumer could decide to use the default set in the library or set the dust threshold themselves (e.g. in a bad fee market, maybe this 1k sats could be considered dust)>

Reply:
So now majorly most of the classes are only using libraries like BigNumber.js for internal implementation only and returning generic outputs to end consumer code ... but getters are returning BN (but as other methods like private one and other are using them so having them output string , will lead to problems in handling implementation in those methods )

Also now DEFAULT_DUST_THRESHOLD supports optional dust threshold provided by user [here](https://github.com/caravan-bitcoin/caravan/commit/77f0cbeb67a274b0aea5a69e87168b9ac1e2304a#diff-47122bb461c82a41c84d8866adf0cb005bf7f573a2afb8558f670a419bed146aR57)